entries/p/plutus.it.json

```diff
@@ -0,0 +1,14 @@
+{
+   "Site Name":{
+      "domain":"plutus.it",
+      "img":"plutus.it.png",
+      "tfa":[
+         "totp"
+      ],
+      "documentation":"https://support.plutus.it/hc/en-us/articles/7203346943517-Account-creation-password-reset-2FA-and-account-block-suspension",
+      "categories":[
+         "banking",
+         "finance"
+      ]
```

Comment:
According to [this article](https://support.plutus.it/hc/en-us/articles/4702798801949), only customers from the following countries are accepted.
```suggestion
      ],
      "regions": [
         "at",
         "be",
         "bg",
         "hr",
         "cz",
         "dk",
         "fi",
         "fr",
         "de",
         "gr",
         "hu",
         "is",
         "ie",
         "it",
         "lv",
         "li",
         "lt",
         "lu",
         "mt",
         "nl",
         "pl",
         "pt",
         "cy",
         "ro",
         "sk",
         "si",
         "es",
         "se"
      ]
```